components/date-picker/internal/DatePickerBase.cs

```diff
@@ -186,7 +193,9 @@ public override CultureInfo CultureInfo
         public string DropdownClassName { get; set; }
 
         [Parameter]
-        public string Format { get; set; }
+        public OneOf<string, string[]> Format { get; set; }
```

Comment:
Only `RangePicker` can apply two `Format`, so please separate them, and do this by another PR.

Reply:
No, multiple formats can apply all pick date components, which inherits DatePickerBase. 
For your approve, have I to separate commits or...? I dont think so, because, I separated commits in one MR

P.S. I added additional examples for format after this message. You can check, DatePicker and RangeDatePicker

Reply:
The `Format` parameter is just a string for display, notinput.

And this PR would be squashed after merged.

Reply:
Okay, I'll separate this PR, but can't undestand why do you have documentation for "format" as "to set the date format, refer to [moment.js](http://momentjs.com/). When an array is provided, all values are used for parsing and first value is used for formatting.".
Do you whant to make two parameters: "DisplayFormat " and "InputFormats"?

Reply:
I think it is should be so, because, if we will have two parameters, we need to set this like
`DisplayFormat : "dd/MM/yyyy"`,
`InputFormat : "dd/MM/yyyy", "ddMMyyyy"`
I think, users want to change one or few digitals in a date, therefore it will be always duplicated for input string as `DisplayFormat`

Reply:
https://github.com/ant-design-blazor/ant-design-blazor/pull/3120